exercises/concept/lasagna/lasagna.cpp

```diff
@@ -0,0 +1,31 @@
+// ovenTime returns the amount in minutes that the lasagna should stay in the
+// oven.
+int ovenTime() {
+    // TODO: Return the correct time.
+    return 0;
+}
+
+/* RemainingOvenTime returns the remaining
+minutes based on the actual minutes already in the oven.
+*/
```

Comment:
Did you purposefully use both `//` comments and `/*` comments for these functions?

Reply:
Yes, so the student can see examples of usage for both types in the file.